index.html

```diff
@@ -0,0 +1,134 @@
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">Logout</a></li>
+    </ul>
+
+    <nav id="navbar" class="header-body">
+        <i id="mobile-menu-close-icon" class="mobile-menu-close-icon fa-solid fa-x"></i>
+        <a class="logo" href="index.html"><img src="./assets/Logopit_1684831391238.png" alt="logo"></a>
+        <ul>
+            <li><a href="about.html">About</a></li>
+            <li><a href="">Programe</a></li>
+            <li><a href="">Join</a></li>
+            <li><a href="">Sponsor</a></li>
+            <li><a href="">News</a></li>
+            <li><a href=""><button class="header-btn">CC Campaign</button></a></li>
+        </ul>
+    </nav>
+
+</header>
+<!-- Header Section Ends Here -->
+
+<main>
+ <!-- Headline Section Starst Here -->
+ <section class="headline">
+    <h6 class="headline__greetings-heading">"Hello! Sharing world"</h6>
+    <div class="headline__main-heading">
+        <h1>GLOBAL TECH SUMMIT 2023</h1>
+        <img src="./assets/orangebg.jpg" alt="">
+    </div>
+    <p class="headline-description">A joyful celebration believing in the value of openness and sharing, creating a 
+        positive change with people from over 80 countries is taking place in October, in India.</p>
+    <h2 class="headline-date">2023.10.15(THU) ~ 16(FRI)</h2>
+    <h6 class="headline-address">@ National Museum of India, Art center Nabi and more</h6>
+</section>
+<!-- Headline Section Ends Here -->
+
+<!-- Program Section Starts Here -->
+<section class="program">
+    <h3 class="program__heading">Main Program</h3>
+    <hr>
+    <ul class="program-items">
+        <li class="program-item">
+            <img src="./assets/program_icon_01.png" alt="">
+            <h4>Lecture</h4>
+            <p>Listen to the speakers from various countries about the messages of sharing and opening.</p>
+        </li>
+        <li class="program-item">
+            <img src="./assets/program_icon_02.png" alt="">
+            <h4>CC Exhibition</h4>
+            <p>Appreciate various creations applying CC license of artists, organized from Art Center Nabi</p>
+        </li>
+        <li class="program-item">
+            <img src="./assets/program_icon_03.png" alt="">
+            <h4>Forum</h4>
+            <p>Have the time to share your thoughts and opinions with experts for each topic.</p>
+        </li>
+        <li class="program-item">
+            <img src="./assets/program_icon_04.png" alt="">
+            <h4>Workshop</h4>
+            <p>Try creating your own work using open source license rather than just watching at it.</p>
+        </li>
+        <li class="program-item">
+            <img src="./assets/program_icon_05.png" alt="">
+            <h4>CC Ignite</h4>
+            <p>get opportunities to network with CC affiliates around the world, also after the summit.</p>
+        </li>
+    </ul>
+    <button>Join Global Tech Summit 2023</button>
+</section>
```

Comment:
- [ ] While you make a great effort😊 I see that you forgot to use the article tags for this section, so it will be better if you use them as this from the requirements, so for this section💯😃, It contains a middle section with a column that includes a section header, a divider, five individual articles, each with an image, title, and description, and a line of text at the bottom.🚀👌

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Your design| 
| --- |
|   ![image](https://github.com/suthish777/global-summit/assets/83360636/644b6660-b781-4e75-9dee-f36cdafeeb96)  |